footer attempt 2

```diff
@@ -0,0 +1,154 @@
+<html>
+    <body style="background-color:lightsteelblue;">
```

Comment:
You have two body and html tags on the document. It should be one. The CSS is also not in a style block. It should be in a separate file.

Reply:
i will try to work on that now

Reply:
The CSS is still in this file.